_episodes/40.md

```diff
@@ -0,0 +1,488 @@
+[388]({{site.url}}docs/current/release/release-388.html#iceberg-connector).
+* [Partition migration]({{site.url}}/docs/current/connector/iceberg.html#alter-table-set-properties)
+([PR](https://github.com/trinodb/trino/pull/12259))
+While Trino was able to read tables with these migrations applied by other query
+engines, this feature allows Trino to write these changes.
+* The following three features are table maintenance commands.
+    * [`optimize`]({{site.url}}/docs/current/connector/iceberg.html#optimize)
+([PR](https://github.com/trinodb/trino/pull/10497)) which is the equivalent to
+the Spark SQL 
+[rewrite_data_files](https://iceberg.apache.org/docs/latest/spark-procedures/#rewrite_data_files).
+    * [`expire_snapshots`]({{site.url}}/docs/current/connector/iceberg.html#expire-snapshots)
+([PR](https://github.com/trinodb/trino/pull/10810)) and uses the equivalent name
+in Spark.
+    * [`remove_orphan_files`]({{site.url}}/docs/current/connector/iceberg.html#remove-orphan-files)
+([PR](https://github.com/trinodb/trino/pull/10810)) and uses the equivalent name
+in Spark.
+* Iceberg v2 support ([PR1](https://github.com/trinodb/trino/pull/11880), [PR2](https://github.com/trinodb/trino/pull/12351), [PR3](https://github.com/trinodb/trino/pull/12749), [PR4](https://github.com/trinodb/trino/pull/11642), [PR5](https://github.com/trinodb/trino/pull/9881), and many more...)
+
+
+Almost every release has some sort of Iceberg improvement around 
+[planning](https://github.com/trinodb/trino/pull/13636) or
+[pushdown](https://github.com/trinodb/trino/pull/13395). If you want all the 
+latest features and performance improvements described here, it's important to
+keep up with the latest Trino version.
+
+## PR 13111: Scale table writers per task based on throughput
+
+This [PR of the episode](https://github.com/trinodb/trino/pull/13111) was 
+contributed by Gaurav Sehgal ([@gaurav8297](https://github.com/gaurav8297)) to 
+enable Trino to automatically scale writers. This PR aims to the number of task
+writers per worker.
+
+You can enable this feature by setting `scale_task_writers` true in your
+configuration. Its initial test results are showing a sixfold speed increase.
+
+Thank you so much to Gaurav and all the reviewers that got this PR through!
+
+## Demo: DML operations on Iceberg using Trino
+
+For this demo of the episode, we use the same schema as the demo we ran in
+[episode 15]({{site.url}}/episodes/15.html), and revise the syntax to
+include new features.
+
+Let's start up a local Trino coordinator and Hive metastore. Clone the 
+repository and navigate to the `iceberg/trino-iceberg-minio` directory. Then
+start up the containers using Docker Compose.
+
+```
+git clone git@github.com:bitsondatadev/trino-getting-started.git
+cd iceberg/trino-iceberg-minio
+docker-compose up -d
+```
+
+Now open up your favorite Trino client and connect it to 
+`jdbc:trino://localhost:8080` to run the following commands:
```

Comment:
We should change this to just localhost:8080 or explain that this URL is the JDBC URL only. Or both. And for example CLI usage is different.

Reply:
we can keep it simple and just remove jdbc